Formula/scrapy.rb

```diff
@@ -0,0 +1,206 @@
+class Scrapy < Formula
+  include Language::Python::Virtualenv
+
+  desc "Web crawling & scraping framework"
+  homepage "https://scrapy.org"
+  url "https://files.pythonhosted.org/packages/eb/b0/7da031ff0ce073839984097724db2c54e8c4b7938acb9ef4691a98620ce7/Scrapy-2.9.0.tar.gz"
+  sha256 "564c972b56e54b83141f395ce3f6a25bfe2093d61d13f9b81d05384e19db98da"
+  license "BSD-3-Clause"
+  head "https://github.com/scrapy/scrapy.git", branch: "master"
+
+  # `pkg-config`, `rust`, and `openssl@1.1` are for cryptography.
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "openssl@1.1"
```

Comment:
Let's wait until after the OpenSSL migration for this, I think.

Reply:
Yes, just running it through CI to see if there are any other issues on older macOS.

Reply:
That's a better idea I didn't think of. I'll open a PR targetting staging.

Reply:
#134777